docker/webserver/nginx_templates/app.conf.template

```diff
@@ -319,6 +319,24 @@ server {
         set $alertmanager http://alertmanager:9093;
         proxy_pass $alertmanager;
     }
+
+    # routinator proxy for internal use on multi instance setups
+    location /routinator/ {
+        include http.headers;
+        include hsts_h3.headers;
+
+        # allow only local connections
```

Comment:
This still requires a 4GB routinator footprint per install .. maybe even optionally open this up even more? (e.g. make batch or dev use single-test Routinator)
I understand doing so creates new potential issues (version and connection wise), and probably it's best running at least one in dev, but batch might use single-test.